authentication/identity_providers/configuring-htpasswd-identity-provider.adoc

```diff
@@ -4,13 +4,34 @@ include::modules/common-attributes.adoc[]
 :context: configuring-htpasswd-identity-provider
 toc::[]
 
-Configure the `htpasswd` identity provider to validate user names and passwords
-against a flat file generated using
-link:http://httpd.apache.org/docs/2.4/programs/htpasswd.html[`htpasswd`].
+[id="identity-provider-overview-{context}"]
+= About identity providers in {product-title}
 
-include::modules/identity-provider-overview.adoc[leveloffset=+1]
+By default, only a `kubeadmin` user exists on your cluster. To specify an
+identity provider, you must create a Custom Resource (CR) that describes
+that identity provider and add it to the cluster.
 
-include::modules/identity-provider-creating-htpasswd-file.adoc[leveloffset=+1]
+[NOTE]
+====
+{product-title} user names containing `/`, `:`, and `%` are not supported.
+====
+
+To define an HTPasswd identity provider you must perform the
+following steps:
+
+. Create a `htpasswd` file to store the user and password information. 
+Instructions are found for
+xref:../../authentication/identity_providers/configuring-htpasswd-identity-provider.adoc#identity-provider-creating-htpasswd-file-linux-{context}[linux]
+and
+xref:../../authentication/identity_providers/configuring-htpasswd-identity-provider.adoc#identity-provider-creating-htpasswd-file-windows-{context}[windows]
+. xref:../../authentication/identity_providers/configuring-htpasswd-identity-provider.adoc#identity-provider-creating-htpasswd-secret-{context}[Create a {product-title} Secret to represent the `htpasswd` file].
```

Comment:
Create a {product-title} -> Create an {product-title}

Reply:
According to the guidance here [1], secret should be lower case

[1] https://github.com/openshift/openshift-docs/blob/enterprise-4.0/contributing_to_docs/term_glossary.adoc#secret

Reply:
Corrected both.